Source/WebInspectorUI/UserInterface/Views/DetailsSection.css

```diff
@@ -45,7 +45,7 @@
     border-top: 1px solid var(--border-color);
 }
 
-.details-section > .header {
+.details-section > .header:not(:empty) {
```

Comment:
I think it might make more sense to just apply the `hidden` class to the header when we don't want it shown, that way we know the entire header will be hidden even if more rules are added in the future that would have applied to it.